### ISSUE TITLE
Request limit on attempts to fill slot when failed

### ISSUE THREAD

**Issue by g23412** [Phr33d0m/NW-Profession-Bot#188]
Example, when trying fill slot, it gives error, because in Leadership still has limit 3 of the same item, instead of trying over and to fill, if it fails, skip that task move to next in list.

also i see in script many times post to console.log but idk how to see this.

**Comment by HenryLoenwind** [comment]
This should not happen. The script doesn't try to start a specific task. Instead it goes through the list of available tasks the server sends and picks one. I'd guess lag on server side; the server has been plenty unstable the last days.

**Comment by g23412** [comment]
yes quite laggy, noticing that if there was a catch for these instances it would prevent the bot from looping endlessly until the next reload or until i happen to notice it stuck.

**Comment by noonereally** [comment]
It supposed to reload the script after 10 attempts (code at about line 2700), to prevent exactly the endless loop.

You can see the log by pressing ctrl+shift+j

**Comment by mystixa** [comment]
Yeah I can confirm that the script is definitely getting stuck occasionally. There is no discernable lag in the past many days that Ive caught it having this issue. I have increased the 'wait time' in the options to see if that would effect anything, and it has not. (not sure if that is waiting in the right area to do any good, but it was a thought.) It does eventually move on, but having 10 failed attempts regularly seems like a bad idea.

I went to look at the code and didnt see anything in the lines around 2700 that really seemed like it was referencing retries. Since you know what you're looking for, any chance of you narrowing that down a bit?

Even if the confusion in the script isnt fixed anyime soon I would like to at least reduce the number of retries down to 2 or 3.
